core/templates/dev/head/pages/exploration_editor/feedback_tab/feedback_tab.html

```diff
@@ -54,7 +54,7 @@ <h2>Exploration Feedback</h2>
     </div>
 
     <div class="row" ng-if="activeThread.messages">
-      <div style="margin-top: 10px; word-break: break-all;" class="col-lg-12 col-md-12 col-sm-12">
+      <div style="margin-top: 10px; word-wrap: break-word;" class="col-lg-12 col-md-12 col-sm-12">
```

Comment:
Can we also avoid inline style attrs in the HTML? Declare a CSS class for it.

/cc @apb7 @lilithxxx this should be a lint check.

Reply:
On it. :)

Reply:
done @seanlip :)

Reply:
@seanlip created issue #6212 for avoiding inline styling. Thanks!